js/wp-seo-term-scraper-305.js

```diff
@@ -377,6 +368,37 @@
 
 			YoastSEO.analyzerArgs.translations = translations;
 		}
+
+		var titlePlaceholder = wpseoTermScraperL10n.title_template;
+		if (titlePlaceholder === '' ) {
+			titlePlaceholder = '%%title%% - %%sitename%%';
+		}
+
+		var metaPlaceholder = wpseoTermScraperL10n.metadesc_template;
+		if (metaPlaceholder === '' ) {
+			metaPlaceholder = 'Modify your meta description by editing it right here';
```

Comment:
This default is already defined in `js/snippetPreview.js`. We don't want to duplicate the default.

So we should only pass the meta placeholder if an actual template is set. Best way is to put the arguments in a variable, for example `args` and only add the `placeholder.metaDesc` value if the metaPlaceholder is not empty.